src/toolboxes/spline/Canvas.tsx

```diff
@@ -314,12 +312,8 @@ export class SplineCanvas extends Component<Props> {
     const clickPoint = canvasToImage(
       x,
       y,
-      this.props.imageWidth,
-      this.props.imageHeight,
-      this.props.scaleAndPan,
-      this.props.canvasPositionAndSize
-    );
-    const annotationData = this.props.annotationsObject.getActiveAnnotation();
+        this.props.imageData.width,
+        this.props.imageData.height,
```

Comment:
I think there's an issue here (looking in the source for this commit, it's not just the diff viewer being weird: [https://github.com/gliff-ai/annotate/blob/026ad0fc6f6d8b19f0dba1970dc9a84dda66ee19/src/toolboxes/spline/Canvas.tsx](https://github.com/gliff-ai/annotate/blob/026ad0fc6f6d8b19f0dba1970dc9a84dda66ee19/src/toolboxes/spline/Canvas.tsx))

Reply:
Sorry, what is the issue?

Reply:
Oh, I think I see... I would ignore that... it doesn't exist in my local copy, must have accidental staged only half a segment